ldi-orchestrator/ldio-application/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/AdapterDebugger.java

```diff
@@ -0,0 +1,27 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldio.config;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.types.LdiAdapter;
+import org.apache.jena.rdf.model.Model;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.stream.Stream;
+
+public class AdapterDebugger implements LdiAdapter {
+	private final Logger logger;
+	private final LdiAdapter adapter;
+
+	public AdapterDebugger(LdiAdapter adapter) {
+		this.logger = LoggerFactory.getLogger(adapter.getClass());
+		this.adapter = adapter;
+	}
+
+	@Override
+	public Stream<Model> apply(Content content) {
+		if (logger.isDebugEnabled()) {
```

Comment:
Is this check necessary? I thought that this logic is already contained in "logger.debug". When you look at AbstractLogger.debug, you see the exact same logic:

`    public void debug(Marker marker, String msg) {
        if (isDebugEnabled(marker)) {
            handle_0ArgsCall(Level.DEBUG, marker, msg, null);
        }
    }`

Reply:
Sure, will remove